.cirrus.yml

```diff
@@ -135,32 +135,32 @@ macos10_task:
 
 # FreeBSD
 freebsd12_task:
-  name: FreeBSD 12.2 x64, DMD ($TASK_NAME_TYPE)
+  name: FreeBSD 13.0 x64, DMD ($TASK_NAME_TYPE)
   freebsd_instance:
-    image_family: freebsd-12-2
+    image_family: freebsd-13-0
     cpu: 4
     memory: 8G
   timeout_in: 60m
   environment:
     OS_NAME: freebsd
-    CI_DFLAGS: -version=TARGET_FREEBSD12
+    CI_DFLAGS: -version=TARGET_FREEBSD13
```

Comment:
Where are these actually used? I grepped for `TARGET_FREEBSD` in dmd/druntime/Phobos, no relevant hits.

Reply:
I belive its just for future convenience, but yeah, probably worth to be on the build system itself rather than the CI config

Reply:
Apparently someone removed the code that used this version condition.

Reply:
This probably also means that the current 12.x pipeline is likely a sham, as the built compiler is default targetting FreeBSD 11.

Reply:
Uh yeah probably. - This probably fell under the carpet with `-target=…` cross-compilation support. `CI_DFLAGS` are most likely not sufficient in that case (only affecting dmd/druntime/Phobos builds, not the default target of the built DMD).

Reply:
That's correct. `git log --pickaxe-regex` tells me https://github.com/dlang/dmd/commit/ee446a03971c99a99de15b28930e205103063260 from #12508 .

Reply:
Cross referencing https://github.com/dlang/dmd/pull/14071